pallets/msa/src/weights.rs

```diff
@@ -76,89 +75,89 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Msa PublicKeyToMsaId (r:1 w:1)
 	// Storage: Msa PublicKeyCountForMsaId (r:1 w:1)
 	fn create() -> Weight {
-		Weight::from_ref_time(19_842_000 as u64)
+		Weight::from_ref_time(20_514_000 as u64)
 			.saturating_add(T::DbWeight::get().reads(3 as u64))
 			.saturating_add(T::DbWeight::get().writes(3 as u64))
 	}
-	// Storage: Msa PayloadSignatureBucketCount (r:1 w:1)
-	// Storage: Msa PayloadSignatureRegistry (r:1 w:1)
+	// Storage: Msa PayloadSignatureRegistryRing (r:1 w:1)
+	// Storage: Msa PayloadSignatureRegistryRingPointer (r:1 w:1)
 	// Storage: Msa PublicKeyToMsaId (r:2 w:1)
 	// Storage: Msa ProviderToRegistryEntry (r:1 w:0)
 	// Storage: Msa CurrentMsaIdentifierMaximum (r:1 w:1)
 	// Storage: Msa PublicKeyCountForMsaId (r:1 w:1)
 	// Storage: Msa DelegatorAndProviderToDelegation (r:1 w:1)
 	// Storage: Schemas CurrentSchemaIdentifierMaximum (r:1 w:0)
 	fn create_sponsored_account_with_delegation(s: u32, ) -> Weight {
-		Weight::from_ref_time(100_556_500 as u64)
-			// Standard Error: 19_778
-			.saturating_add(Weight::from_ref_time(120_447 as u64).saturating_mul(s as u64))
+		Weight::from_ref_time(102_877_072 as u64)
+			// Standard Error: 18_917
+			.saturating_add(Weight::from_ref_time(78_078 as u64).saturating_mul(s as u64))
 			.saturating_add(T::DbWeight::get().reads(9 as u64))
 			.saturating_add(T::DbWeight::get().writes(6 as u64))
 	}
 	// Storage: Msa PublicKeyToMsaId (r:1 w:0)
 	// Storage: Msa DelegatorAndProviderToDelegation (r:1 w:1)
 	fn revoke_delegation_by_provider() -> Weight {
-		Weight::from_ref_time(23_624_000 as u64)
+		Weight::from_ref_time(23_611_000 as u64)
 			.saturating_add(T::DbWeight::get().reads(2 as u64))
 			.saturating_add(T::DbWeight::get().writes(1 as u64))
 	}
-	// Storage: Msa PayloadSignatureBucketCount (r:1 w:1)
-	// Storage: Msa PayloadSignatureRegistry (r:2 w:2)
+	// Storage: Msa PayloadSignatureRegistryRing (r:2 w:2)
+	// Storage: Msa PayloadSignatureRegistryRingPointer (r:1 w:1)
 	// Storage: Msa PublicKeyToMsaId (r:2 w:1)
 	// Storage: Msa PublicKeyCountForMsaId (r:1 w:1)
 	fn add_public_key_to_msa() -> Weight {
-		Weight::from_ref_time(147_786_000 as u64)
+		Weight::from_ref_time(155_708_000 as u64)
```

Comment:
Does appear there is a slight increase in base weight, but this is also within the range of variability of weight generation.